internal/tfutils/tfutils.go

```diff
@@ -120,6 +124,13 @@ func setBool(field reflect.Value, tagValue string, out map[string]string) {
 	out[tagValue] = fmt.Sprintf("%v", fieldVal)
 }
 
+func setStringSlice(field reflect.Value, tagValue string, out map[string]string) {
+	if !field.IsNil() {
+		valueString := fmt.Sprintf("%v", strings.Join(field.Interface().([]string), ","))
+		out[tagValue] = valueString
+	}
+}
```

Comment:
Could you please add a small test case to capture the intended behaviour of this?

Reply:
Added test case for this scenario